python/ray/tests/test_runtime_env.py

```diff
@@ -33,7 +33,7 @@
 
 
 def test_get_wheel_filename():
-    ray_version = "2.0.0.dev0"
+    ray_version = "1.12.0rc0"
```

Comment:
Are these ok to be changed? Let's wait for test results. Maybe they should use an obviously fake version string instead.

Reply:
Reverted.